client/view/BrowserStats/browserStats.js

```diff
@@ -1,26 +1,29 @@
 import wikipediaLinks from '../../data/wikipedia-links.js'
 
-const WIKIPEDIA_URL = 'https://en.wikipedia.org/wiki/'
+export function updateRegionCoverageCaption(coverage) {
+  let element = document.querySelector('[data-id=region_coverage]')
+  element.innerHTML = coverage + '%'
+}
 
-export function updateGlobalCoverageBar(data) {
-  let element = document.querySelector('[data-id=global-coverage-bar]')
+export function updateRegionCoverageBar(data) {
+  let element = document.querySelector('[data-id=region_coverage_bar]')
   element.innerHTML = ''
   data.forEach(item => {
     let itemElem = document.createElement('li')
-    itemElem.classList.add('BrowsersStat__globalCoverageBarItem')
+    itemElem.classList.add('BrowsersStat__regionCoverageBarItem')
     itemElem.style.setProperty('--proportion', item.coverage)
     itemElem.style.setProperty('--alpha', 1 - 1 / item.coverage)
     if (item.coverage > 10) {
       itemElem.innerHTML = item.name
-      itemElem.classList.add('BrowsersStat__globalCoverageBarItem--texted')
+      itemElem.classList.add('BrowsersStat__regionCoverageBarItem--texted')
     }
     element.appendChild(itemElem)
   })
 }
 
-const statsPlaceholder = document.querySelector('.BrowsersStat__placeholder')
-
 export function hideStatsPlaceholder() {
+  let statsPlaceholder = document.querySelector('.BrowsersStat__placeholder')
```

Comment:
Should we use `data-id` for every query from JS?

Reply:
Oh, they were still there

sure)